lib/CMakeLists.txt

```diff
@@ -287,6 +287,22 @@ if(QUDA_INTERFACE_TIFR OR QUDA_INTERFACE_ALL)
   target_compile_definitions(quda PUBLIC BUILD_TIFR_INTERFACE)
 endif(QUDA_INTERFACE_TIFR OR QUDA_INTERFACE_ALL)
 
+if(QUDA_CONTRACT)
```

Comment:
looks like the target_compile_definitions should not be added back. IIRC @maddyscientist removed those in favor of a configured header.

Reply:
Ah, good catch @mathiaswagner

Reply:
If I understand correctly, it's not just a problem for this pr to fix but a general issue. Right?

Reply:
Resolved with [daa6244](https://github.com/lattice/quda/pull/1312/commits/daa6244e041e55034a1adbe8cc0a47bd86b790b6)